tests/bluetooth/host/buf/bt_buf_get_cmd_complete/testcase.yaml

```diff
@@ -3,18 +3,23 @@ common:
 tests:
   bluetooth.host.bt_buf_get_cmd_complete.default:
      type: unit
+     extra_configs:
+      - CONFIG_ASSERT=n
```

Comment:
Is there an idea behind adding that extra config `CONFIG_ASSERT=n`? Is it required?

Reply:
yes, else it will try to link a partial assert from Zephyr which fails during linking.

This was discussed elsewhere, and there will come a PR soon that most likely will disable asserts default on unittesting board.

Reply:
this is actually interresting.

The initial PR for unit_testing board already introduces `CONFIG_ASSERT=n`:
https://github.com/zephyrproject-rtos/zephyr/blob/a5ee862ff0b857ff98357bf5f4e9cd1568c178f3/subsys/testsuite/boards/unit_testing/unit_testing/unit_testing_defconfig#L10-L12

so this shouldn't be needed, so something might be cancelling out the default.
Will investigate.

Reply:
did some testing, and I can't reproduce the original compilation error related to includes and asserts anymore, so remove this part.